Packs/GoogleCloudLogging/Integrations/GoogleCloudLogging/GoogleCloudLogging.py

```diff
@@ -0,0 +1,263 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+import urllib3
+from typing import Dict, Any
+from GSuiteApiModule import *  # noqa: E402
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+SCOPES = ['https://www.googleapis.com/auth/logging.read']
+MAX_LIMIT = 1000
+
+
+''' HELPER FUNCTIONS '''
+
+
+def prepare_gsuite_client() -> GSuiteClient:
+    """
+    Creates a client.
+
+    Args:
+        request_body (dict): The request body.
+        client (GSuiteClient): GSuiteClient client.
+
+    Returns:
+        The request response.
+    """
+    credentials_json = demisto.params().get('credentials', {}).get('password')
+    verify_certificate = not demisto.params().get('insecure', False)
+    proxy = demisto.params().get('proxy', False)
+    headers = {'Content-Type': 'application/json'}
+    try:
+        service_account_json = GSuiteClient.safe_load_non_strict_json(credentials_json)
+        client = GSuiteClient(service_account_json,
+                              base_url='https://logging.googleapis.com/', verify=verify_certificate, proxy=proxy,
+                              headers=headers)
+    except Exception as e:
+        raise e
+    return client
+
+
+def get_entries_request(client: GSuiteClient, request_body: dict) -> dict:
+    """
+    Gets a request body and execute the request.
+
+    Args:
+        request_body (dict): The request body.
+        client (GSuiteClient): GSuiteClient client.
+
+    Returns:
+        The request response.
+    """
+    client.set_authorized_http(scopes=SCOPES)
+    return client.http_request(
+        url_suffix='v2/entries:list', method='POST', body=request_body
+    )
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(params: dict) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        prepare_gsuite_client()
+        message = 'ok'
+    except Exception as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
```

Comment:
The error message is still incorrect, you don't have API key here. 
Fix the message accordingly